content/docs/details/scaffolder/index.md

```diff
@@ -0,0 +1,403 @@
+---
+title: Scaffolder
+publishedDate: '2022-05-16T10:53:00.0Z'
+description: Initialize projects and trigger automation from within Backstage. You can prompt parameter inputs and combine scaffolder actions in order to build a scaffolder template.
+---
+
+## Overview
+The Roadie Backstage scaffolder is a feature that allows you to define software templates to create new software projects, update existing ones or simply perform repeated tasks in a consistent manner.
```

Comment:
Does it actually allow you to update existing projects?

Reply:
Yes. you can create pull requests against existing projects. The action `publish:github:pull-request` is used by one of our customers to add resources to an existing terraform project based on a template and user input.

Reply:
Let me know if this is ok?